src/cycax/cycad/assembly.py

```diff
@@ -70,24 +89,6 @@ def add(self, part: CycadPart):
             part: this in the part that will be added to the assembly.
         """
 
-        dir_name = f"{os.getcwd()}/{part.part_no}"
-        if not os.path.exists(dir_name):
-            os.mkdir(dir_name)
-
-        with open("./temp.json", "w") as jsonfile:
-            json.dump(part.export(), jsonfile, indent=4)
-
-        JSONname = f"{dir_name}/{part.part_no}.json"
-        if os.path.exists(JSONname):  # checking if the file is already in the directory.
-            current = os.stat(JSONname).st_size
-            new = os.stat("./temp.json").st_size
-            if new >= current:  # addind the file if it is bigger.
```

Comment:
I do like this, but I did not maintain it. I had a case where I removed features from an objects and I could not figure out why it did not update.

We have a problem if the same part is defined twice and manipulated differently, but then it should have been different parts. Or later we can add a part merge that will merge the features from two parts to make a universal part (as was done with the small storage sleds).